tests/test_helpers.py

```diff
@@ -98,8 +98,7 @@ async def goto_url(
         websocket,
         context_id: str,
         url: str,
-        wait: Literal["none", "interactive",
-                      "complete"] = "interactive") -> dict:
+        wait: Literal["none", "interactive", "complete"] = "complete") -> dict:
```

Comment:
We should wait all events to complete else we end up listening to the `load` when we subscribe.